fabric/src/main/java/dev/architectury/mixin/fabric/client/MixinGameRenderer.java

```diff
@@ -39,7 +39,7 @@
 import java.util.List;
 import java.util.function.Consumer;
 
-@Mixin(GameRenderer.class)
+@Mixin(value = GameRenderer.class, priority = 1100)
```

Comment:
What was the purpose of changing the priority here?

Reply:
I talk about why in https://github.com/architectury/architectury-api/pull/167/commits/3d2a81f80745eca08ec1957005062a1d1f19f429

Reply:
If it's <= 1000, it crashes because it performs a modification that is incompatible with the Fabric API modification later on.